File: InschrijvingenApp/Entities/Kind.cs
using InschrijvenPietieterken.Entities;
using System;

namespace InschrijvingPietieterken.Entities
{
    public class Kind : EntityBase
    {
        public Kind()
        {
            Persoon = new Persoon();
        }

        public Persoon Persoon { get; set; }
        public DateTime GeboorteDatum { get; set; }
        public string BroersZussen { get; set; } //seperate sibligs with ;
        public bool ZelfNaarHuis { get; set; }
        public string AfhaalPersoon { get; set; }

    }
}

File: InschrijvingenApp/ClientApp/app/components/models/kind.ts
import { Persoon } from "./persoon";

export class Kind {

    constructor() {
        this.persoon = new Persoon();
        this.geboortedatum = new Date("2014-01-01");
    }

    public persoon: Persoon;
    public geboortedatum: Date;
    public broersZussen: string = '';
    public zelfNaarHuis: boolean = false;
    public afhaalPersoon: string = '';

}